<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GPushOk">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ok.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="GInputPairs">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;
border: 2px solid rgb(80,80,80);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="GNameMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 11pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Generate file</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="HNameMain">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 11pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Hash table file</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="HPushOk">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ok.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="GInputLength">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>105</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;
border: 2px solid rgb(80,80,80);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="GNamePairs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Pairs num:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GNameLength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Key length:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GNameFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="GInputFile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;
border: 2px solid rgb(80,80,80);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="HNameFile">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Input file:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="VLine">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>25</y>
      <width>3</width>
      <height>310</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="HLine">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>160</y>
      <width>600</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="MNameMain">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 11pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Graphmaker</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="GCheckWorst">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Worst case for HFunc</string>
    </property>
   </widget>
   <widget class="QLabel" name="MNameMin">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>255</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="MInputMin">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>255</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;
border: 2px solid rgb(80,80,80);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="MNameMax">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>255</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="MInputMax">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>255</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;
border: 2px solid rgb(80,80,80);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="HNameHFunc">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>HFunc number:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="HInputFile">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;
border: 2px solid rgb(80,80,80);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MPushOk">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ok.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinHFunc">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>43</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QCustomPlot" name="outputPlot" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>1220</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QTextEdit" name="outputText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1220</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);
color: white;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="HInputOut">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>105</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 4px 5px 0px 7px;
border: 2px solid rgb(80,80,80);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="HNameOut">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>105</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Output file:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="HNameLength">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>105</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Factor:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinFactor">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>105</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="MTestPairs">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Pairs num</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MTestKey">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Key length</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MTestFactor">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Factor</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinStep">
    <property name="geometry">
     <rect>
      <x>842</x>
      <y>255</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="HNameStep">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>255</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="MLayout">
     <item>
      <widget class="QRadioButton" name="MTestSum">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
       </property>
       <property name="text">
        <string>Iterations sum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="MTestMax">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
       </property>
       <property name="text">
        <string>Longest insert</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="OutLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>1220</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OutLayout">
     <item>
      <widget class="QRadioButton" name="OutSetPlot">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
       </property>
       <property name="text">
        <string>Plot output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="OutSetText">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(235, 235, 235);</string>
       </property>
       <property name="text">
        <string>Text output</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="OutSaveFile">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>380</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/savef.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>GPushOk</zorder>
   <zorder>GInputPairs</zorder>
   <zorder>GNameMain</zorder>
   <zorder>HNameMain</zorder>
   <zorder>HPushOk</zorder>
   <zorder>GInputLength</zorder>
   <zorder>GNamePairs</zorder>
   <zorder>GNameLength</zorder>
   <zorder>GNameFile</zorder>
   <zorder>GInputFile</zorder>
   <zorder>HNameFile</zorder>
   <zorder>VLine</zorder>
   <zorder>HLine</zorder>
   <zorder>MNameMain</zorder>
   <zorder>GCheckWorst</zorder>
   <zorder>MNameMin</zorder>
   <zorder>MInputMin</zorder>
   <zorder>MNameMax</zorder>
   <zorder>MInputMax</zorder>
   <zorder>HNameHFunc</zorder>
   <zorder>HInputFile</zorder>
   <zorder>MPushOk</zorder>
   <zorder>spinHFunc</zorder>
   <zorder>HInputOut</zorder>
   <zorder>HNameOut</zorder>
   <zorder>HNameLength</zorder>
   <zorder>spinFactor</zorder>
   <zorder>MTestPairs</zorder>
   <zorder>MTestKey</zorder>
   <zorder>MTestFactor</zorder>
   <zorder>spinStep</zorder>
   <zorder>HNameStep</zorder>
   <zorder>outputPlot</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>OutLine</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>OutSaveFile</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
